src/AirshipBindings.NETStandard/AirshipBindings.NETStandard/Airship.cs

```diff
@@ -7,6 +7,9 @@
 
 namespace UrbanAirship.NETStandard
 {
+    public delegate void DeepLinkHandler(string deepLink);
```

Comment:
I think part of what has been causing problems is these custom delegate types. the fact that there were multiple declarations of InboxHandler before is probably why there were type errors at runtime.

We don't need to reinvent the wheel here. EventHandler is a perfectly idiomatic delegate type to use for events (see MessagePage for an example).

Reply:
You can also use Action as a delegate, since it's already typed that way in the standard library. That would allow you to define events that use lambdas as the delegate part of the declaration, e.g.

```
public event Action OnMessageCenterUpdated;

...

OnMessageCenterUpdated += () => { 
    // do something
};
```

Or similarly for deep links:

```
public event Action<string> OnDeepLinkReceived
```

The advantage of this is that you don't need the sender or event args parameters that EventHandler uses, and you don't need to declare any extra types (such as custom subclasses of EventArgs) in order to pass custom event data. Which one you pick is up to you.

Reply:
Actually forget the deep link stuff for now, I'll make a separate ticket for it. Just focus on inbox events.

Reply:
Actually, don't worry about changing the deep link stuff. I'll make a separate ticket for that. Just focus on inbox events for now.